core/modules/installer/installer.browser.inc

```diff
@@ -22,11 +22,11 @@ function _installer_browser_is_project_enabled($type, $name) {
       break;
     case 'theme':
       $themes = list_themes();
-      return isset($themes[$name]);
+      return (isset($themes[$name]) && $themes[$name]->status == 1);
       break;
     case 'layout':
-      $layout = layout_load($name);
-      return !empty($layout);
+      $excluded_templates = config_get('layout.settings', 'excluded_templates');
+      return (!empty(layout_get_layout_info($name)) && !in_array($name, $excluded_templates));
```

Comment:
I think this line is causing the error on PHP 5.3. Prior to version 5.5 I think, you can't use `empty()` to check a value returned by a function, it can only check variables. See http://php.net/empty.

I think if we change this to a simple loose check this would work better:
```
return (layout_get_layout_info($name) && !in_array($name, $excluded_templates));
```